### PR TITLE
init shortcuts : add action of add/remove shortcut

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

**Review by mbarbet — APPROVED** [review]

File: src/app/modules/analytics-config/components/widget-edition/widget-edition.component.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  /** changes icon/text according to add/remove the shortcut */
+  private setPinTemplate(usage: WidgetUsage) {
+    if (!usage || usage === 'analytics') {
+      this.pinIconName = 'add_circle_outline';
+      this.pinText = 'Pin widget';
+    } else {
+      this.pinIconName = 'remove_circle_outline';
+      this.pinText = 'Unpin widget';
```
Comment (sebbousquet) [inline src/app/modules/analytics-config/components/widget-edition/widget-edition.component.ts:76]: Add the `marker` method for translation

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -818,5 +818,6 @@
 	"desc": "Descending sort",
 	"compute metric description": " • Compute a metric on the chosen field. \n • Optionally, apply an ARLAS filter on the computed metric.",
 	"metric": "Metric",
-	"Cannot save the preview": "Cannot save the preview"
+	"Cannot save the preview": "Cannot save the preview",
+	"Pin widget" : "Add to filters band"
```
Comment (sebbousquet) [inline src/assets/i18n/en.json:822]: Missing `Unpin widget` translation

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -820,5 +820,6 @@
 	"desc": "Tri descendant",
 	"compute metric description": " • Calculez une métrique sur le champ choisi. \n • Optionallement, appliquez un filtre ARLAS sur la métrique calculée. ",
 	"metric": "Métrique",
-	"Cannot save the preview": "Impossible de sauvegarder l'aperçu"
+	"Cannot save the preview": "Impossible de sauvegarder l'aperçu",
+	"Pin widget" : "Ajouter à la barre des filtres"
```
Comment (sebbousquet) [inline src/assets/i18n/fr.json:824]: Missing `Unpin widget` translation

**Review by sebbousquet — APPROVED** [review]
LGTM
Still missing the `marker` function but not mandatory

File: src/app/modules/analytics-config/components/widget-edition/widget-edition.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/modules/analytics-config/components/widget-edition/widget-edition.component.ts:76]: I'll add it in the next pr

**Review by mbarbet — APPROVED** [review]

**Review by QuCMGisaia — APPROVED** [review]

**Pull request merged**